ra_aid/agent_utils.py

```
@@ -359,7 +363,12 @@ def run_research_agent(
         logger.warning(f"Failed to get project info: {e}")
         formatted_project_info = ""
 
+    current_date = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)